setup.py

```diff
@@ -8,6 +8,10 @@
 with codecs.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
```

Comment:
Right! I have just added MANIFEST.in to make requirements.txt part of the package and now pip should not complain. At least that what we do in https://github.com/incf-nidash/nidmresults-fsl and pip install works. Is that fine?